stable/datacube-dev-env/templates/install-postgis.yaml

```diff
@@ -0,0 +1,88 @@
+      labels:
+        heritage: {{.Release.Service | quote }}
+        release: {{.Release.Name | quote }}
+        chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+        {{- if .Values.database.create }}
+        - name: create-db
+          image: "{{ .Values.createdb.image.repository }}:{{ .Values.createdb.image.tag }}"
+          args: [ {{- range .Values.createdb.dockerArgs }} {{ . | quote }}, {{ end -}} ]
+          imagePullPolicy: {{ .Values.createdb.image.pullPolicy }}
+          env:
+            - name: DB_HOSTNAME
+              value: {{ .Values.database.host | quote }}
+            - name: DB_PORT
+              value: {{ .Values.database.port | quote }}
+            - name: DB_DATABASE
+              value: {{ .Values.database.database | quote }}
+            - name: DB_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.database.existingSecret }}
+                  key: postgres-username
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.database.existingSecret }}
+                  key: postgres-password
+            - name: ADMIN_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.database.adminSecret }}
+                  key: postgres-username
+            - name: ADMIN_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.database.adminSecret }}
+                  key: postgres-password
+        {{- end }}
+        - name: install-postgis-extension
+          image: "{{ .Values.setup.image.repository }}:{{ .Values.setup.image.tag }}"
+          args: [ {{- range .Values.setup.dockerArgs }} {{ . | quote }}, {{ end -}} ]
+          imagePullPolicy: {{ .Values.setup.image.pullPolicy }}
+          env:
+          - name: DB_HOSTNAME
+            value: {{ .Values.database.host | quote }}
+          - name: DB_PORT
+            value: {{ .Values.database.port | quote }}
+          - name: DB_DATABASE
+            value: {{ .Values.database.database | quote }}
+          - name: DB_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.database.existingSecret }}
+                key: postgres-username
+          - name: DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.database.existingSecret }}
+                key: postgres-password
+          - name: ADMIN_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.database.adminSecret }}
+                key: postgres-username
+          - name: ADMIN_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.database.adminSecret }}
+                key: postgres-password
+      restartPolicy: Never
```

Comment:
Potsgis extension shouldn't be required either, (they can just run the command if needed)

Reply:
Postgis extension template file removed